lib/minify-source-tree.js

```diff
@@ -30,6 +30,7 @@ module.exports = function minifySourceTree(tree, options) {
     sourceMapConfig: {
       enabled: options.enableSourceMaps
     },
+    negate_iife: options.negateIIFE, // jshint ignore:line
```

Comment:
Can you add `// jshint ignore:line` here?  The underscore case is causing a JSHint failure.